### PR TITLE
Performed a copy edit (fixed typos, grammar, and formatting) in all pages of the docs

### PR DESCRIPTION
**Pull request by sallylanda6**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

I read all pages in the docs and made copy edits per Srini's request, including but not limited to:

- Fixed typos
- Fixed grammar
- Added missing punctuation
- Updated formatting to adhere to Axway standards
- Resolved Markdown errors 
- Added text in between stacked headings where it was missing
- Changed instructional sections to use numbered steps, where possible 
- Spelled out contractions

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

File: content/en/docs/Configuration/Certificate Management/mtls.md [hunk 1]
```diff
@@ -8,35 +8,37 @@ date: 2021-09-02
 
 ## Context
 
-Mutual authentication is required for most of API developed for Open Banking.
+Mutual authentication is required for most APIs developed for Open Banking.
 
-According to the Open Banking Specification, the MTLS is required for both components : Cloud Entity and the API Gateway Listener. Here is a diagram that explains how it's working.
+According to the Open Banking Specification, the MTLS is required for the Cloud Entity and API Gateway Listener components. See the diagram which explains how it is working.
 ![MTLS diagram](/Images/mtls.png)
```
Comment (sriniip) [inline content/en/docs/Configuration/Certificate Management/mtls.md:14]: According to the Open Banking Specification, Mutual Transport Layer Security (MTLS) based client connection is required for the Cloud Entity and API Gateway Listener components. See the diagram for details on the MTLS setup.

File: content/en/docs/Configuration/Certificate Management/mtls.md [hunk 2]
```diff
@@ -8,35 +8,37 @@ date: 2021-09-02
...
 
-Others possibilities are :
+Others possibilities are:
 
 * Use a component in front of the Kubernetes cluster to support the MTLS termination. In this condition, Axway recommands to have a component nearest the Kubernetes cluster.
 
 * Replace the nginx ingress controller by another ingress controller that supports the required features.
 
-See the required features the ingress controler in [Deployment > Prerequisites](/docs/deployment/prerequisites)
+Refer to the required features the ingress controller in [Deployment - Prerequisites](/docs/deployment/prerequisites).
```
Comment (sriniip) [inline content/en/docs/Configuration/Certificate Management/mtls.md:28]: Refer to the required features for the ingress controller in

File: content/en/docs/Configuration/Certificate Management/mtls.md [hunk 3]
```diff
@@ -49,15 +51,16 @@ Note : The target architecture is that API-Gateway must route the request to ACP
...
 ```bash
-openssl genrsa -out ca1.key 2048openssl req -new -x509 -days 3650 -key ca1.key -subj "/C=BR/ST=São Paulo/L=São Paulo/O=Axway/CN=Axway Root CA" -out ca1.crtopenssl genrsa -out ca2.key 2048openssl req -new -x509 -days 3650 -key ca2.key -subj "/C=BR/ST=São Paulo/L=São Paulo/O=Axway/CN=Axway Root CA" -out ca2.crt
+openssl genrsa -out ca1.key 2048openssl req -new -x509 -days 3650 -key ca1.key -subj "/C=BR/ST=São Paulo/L=São Paulo/O=Axway/CN=Axway Root CA" -out ca1.crtopenssl genrsa -out ca2.key 2048openssl req -new -x509 -days 3650 -key ca2.key -subj "/C=BR/ST=São Paulo/L=São Paulo/O=Axway/CN=Axway Root CA" -out ca2.crt
 ```
 
-### Create certificates for the Third Party Provider App (Client Certificates for each TPP) 
+### Create certificates for the Third Party Provider (TPP) App (Client Certificates for each TPP)
 
-Each certificate must have one key and signed with a root CA previously created. These configuration files below are provided as example.
+Each certificate must have one key and signed with a root CA previously created. These configuration files below are provided as examples.
```
Comment (sriniip) [inline content/en/docs/Configuration/Certificate Management/mtls.md:59]: Each certificate must have one key that is signed with a previously created root certificate authority.

File: content/en/docs/Configuration/amplify-agents.md [hunk 4]
```diff
@@ -12,43 +12,40 @@ The Amplify agents are software applications that run on your host. The agents a
...
+    * **Method**: select **Client Certificate**.
+    * **Credentials**: Select **Platform-generated key pair** or **Provide public key** if you have your own certificate.
+6. You can add one or more organizations and teams, but the default options would work.
+7. Click **Save**.
+8. A pop-up window will appear with the private key. Download it and save a copy as it will be used in the `values.yaml` file.
 
 ![Service Account Private key window in Amplify](/Images/agents/service-account-privatekey.PNG)
 
-The service account client id, name, public and private key  will be used in the values.yaml file in the following parameters:
+The service account client id, name, public, and private key will be used in the `values.yaml` file in the following parameters:
```
Comment (sriniip) [inline content/en/docs/Configuration/amplify-agents.md:48]: I always have trouble getting the voice of the sentence right. Should this be in the active voice or does it matter..

File: content/en/docs/Configuration/api-manager.md [hunk 5]
```diff
@@ -18,43 +18,45 @@ This page is under development
...
 
-* change API logo
-* change API summary
-* change API documentation
+* API logo
+* API summary
+* API documentation
 
-These Frontend API changes can be done only by unpublishing the API first:
+These frontend API changes can be done only by unpublishing the API first.
```
Comment (sriniip) [inline content/en/docs/Configuration/api-manager.md:40]: Should this be changed to say -  Unpublish the front-end API before applying changes to the API?

File: content/en/docs/Configuration/customization.md [hunk 6]
```diff
@@ -9,39 +9,38 @@ date: 2021-09-02
 
 ## Solution Branding
 
-You can easily apply your branding on most solution user interface.
-This is usually change logos and style (color palette, font, etc.)
+You can easily apply your branding on most of the solution's user interface, such as changing the logos and the style (color palette, font, and so on.)
 
-Here are the main items you can consider for branding:
+The following are the main items you can consider for branding:
```
Comment (sriniip) [inline content/en/docs/Configuration/customization.md:14]: Consider the following for branding changes

File: content/en/docs/Configuration/customization.md [hunk 7]
```diff
@@ -9,39 +9,38 @@ date: 2021-09-02
...
     ![Welcome email](/Images/welcome-email.png)
-* Logo and style of Developer Portal: [Developer portal > Template customization](/docs/configuration/portal/#template-customization)
-* Logo and style of Analytics homepage. Refer to deployment option for this component : [Customize Analytics Helm chart](/docs/deployment/installation/analytics/#customize-analytics-helm-chart)
+* Developer Portal logo and style: [Developer portal - Template customization](/docs/configuration/portal/#template-customization).
+* Analytics homepage logo and style. Refer to the deployment option for this component: [Customize Analytics Helm chart](/docs/deployment/installation/analytics/#customize-analytics-helm-chart).
 
 ## Consent page
 
-Each Axway Open Banking customer would need to customize the Consent page that is embedded in the solution.
+Each Axway Open Banking customer needs to customize the Consent page that is embedded in the solution.
```
Comment (sriniip) [inline content/en/docs/Configuration/customization.md:25]: The Consent page embedded in the solution is provided for demonstration purposes only. Each customer should customize the consent page to match their target application's requirements.

File: content/en/docs/Configuration/customization.md [hunk 8]
```diff
@@ -51,67 +50,65 @@ docker push  <your-docker-repo>/open-banking-consent-page:<image-tag>
...
 
 The consent page should now reflect your changes on your Kubernetes environment.
 
-The sections below detail how to customize the consent page in the docker image project.
+The sections below details how to customize the consent page in the Docker image project.
 
 ### Consent Page template files
 
-Consent page are generated based on template files, that you can customize.
+The consent pages are generated based on template files that you can customize.
```
Comment (sriniip) [inline content/en/docs/Configuration/customization.md:65]: The consent pages are generated based on customizable template files.

File: content/en/docs/Configuration/amplify-agents.md (hunk 4 above)
Comment (sallylanda6) [reply content/en/docs/Configuration/amplify-agents.md:48]: As discussed, I'll update voice in another PR.

**Pull request merged**